### PR TITLE
Bring back the context menu

### PR DESCRIPTION
**Pull request by tobiasdiez**
before | after
------------ | -------------
before | image

Removed a few commands that are not needed often and thus don't need to be placed that prominently.
- Print entry preview: available through entry preview
- All commands related to marking: marking is not yet reimplemented
- Set/clear/append/rename fields: available through Edit menu
- Manage keywords: available through Edit menu
- Copy linked files to folder:  available through File menu

Changes in the architecture:
- Previously, an `Action` defined both the visual properties (text, icons, ...) as well as the behavior. These actions are usually defined in the base panel (or main frame).
- Now the visual elements are defined by an `ActionFX` (to be renamed to `Action` as soon as the old class can be removed) and the behavior by implementing the `Command` interface. This separation makes it possible to reuse the visual aspects without the ties to the behavior.
- In the long term, the commands should be defined outside of the main big classes. This is not done in this PR, because then the scope and size would explode. The extraction is best done peu à peu.

----

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Screenshots added (for bigger UI changes)
- [ ] Manually tested changed features in running JabRef
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

File: src/main/java/org/jabref/gui/actions/OldCommandWrapper.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * This wraps the old Swing commands so that they fit into the new infrastructure.
+ * In the long term, this class should be removed.
+ */
+public class OldCommandWrapper extends CommandBase {
+
+    private static final Log LOGGER = LogFactory.getLog(OldCommandWrapper.class);
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/actions/OldCommandWrapper.java]: Please use the new logging     private static final Logger LOGGER = LoggerFactory.getLogger(OldCommandWrapper.class);

**Comment by Siedlerchr** [comment]
> Copy linked files to folder: moved to Tools menu
Already presend in the File menu
Open File PDF icon is missing

File: src/main/java/org/jabref/gui/actions/OldCommandWrapper.java (hunk 1 above)
Comment (tobiasdiez) [reply src/main/java/org/jabref/gui/actions/OldCommandWrapper.java]: Your PR is not yet merged into `javafxTable` and I prefer to do this as soon as all the other maintable PRs are merged

**Comment by tobiasdiez** [comment]
@Siedlerchr Thank you for the feedback. I implemented your suggestions accordingly.

File: src/main/java/org/jabref/gui/actions/OldCommandWrapper.java (hunk 1 above)
Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/actions/OldCommandWrapper.java]: Okay, just as a hint to make work then easier. Slf4j has a nice migrator tool (inlcuded in the zip download) https://www.slf4j.org/migrator.html to replace all occurences. Used that for the migration, too.

**Comment by koppor** [comment]
Can't we have an "Edit" submenu here containing all the edit actions available?

For me, this conforms to "Lernförderlichkeit" and "Erwartungskonformität" from [EN ISO 9241-110 Grundsätze der Dialoggestaltung](https://de.wikipedia.org/wiki/EN_ISO_9241#EN_ISO_9241-110_Grunds%C3%A4tze_der_Dialoggestaltung). Especially the "Erwawrungskonformität", because I expect that all actions available for the current entry are available in the context menu.

**Pull request merged**